src/routes/Categories.js

```diff
@@ -1,7 +1,14 @@
-const Categories = () => (
-  <div className="container">
-    <button type="button">Check status</button>
-  </div>
-);
+import { useDispatch } from 'react-redux';
+import { checkStatus } from 'redux/categories/categoriesSlice';
+
+const Categories = () => {
+  const dispatch = useDispatch();
+
+  return (
+    <div className="container">
+      <button type="button" onClick={() => dispatch(checkStatus())}>Check status</button>
+    </div>
```

Comment:
- [ ] Please your `Check Status` button does not dispatches the Under Construction action, try to fix that.

Reply:
Hi @CollinsTatang ,
Thanks for the feedback. Could you please help me with more details on this, I really do not understand what I am supposed to fix. Thank you

Reply:
Can we have a call

Reply:
yes, thank you

Reply:
Hello @mrprotocoll 
When a User Click the `Check Status` button under categories nothing happened
It supposes to display `Under Construction `

Reply:
> yes, thank you

Send a zoom call link

Reply:
Alright, Thank you @CollinsTatang , i will make the changes and request a new review